src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/FunctionPointerCallingConventionClassifier.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Classification.Classifiers;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.LanguageServices;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Classification
+{
+    internal class FunctionPointerCallingConventionClassifier : AbstractSyntaxClassifier
+    {
+        public override ImmutableArray<Type> SyntaxNodeTypes { get; } = ImmutableArray.Create(
+            typeof(FunctionPointerUnmanagedCallingConventionSyntax));
+
+        public override void AddClassifications(
+            SyntaxNode syntax,
+            SemanticModel semanticModel,
+            ClassificationOptions options,
+            ArrayBuilder<ClassifiedSpan> result,
+            CancellationToken cancellationToken)
+        {
+            var callingConventionSyntax = (FunctionPointerUnmanagedCallingConventionSyntax)syntax;
+            if (callingConventionSyntax.IsSpecialUnmanagedCallingConvention())
+            {
+                result.Add(new ClassifiedSpan(callingConventionSyntax.Span, ClassificationTypeNames.Keyword));
+                return;
+            }
+
+            var type = semanticModel.Compilation.UnmanagedCallingConventionType(callingConventionSyntax.Name.ValueText);
+            if (type is not null)
+            {
+                result.Add(new ClassifiedSpan(callingConventionSyntax.Name.Span, ClassificationTypeNames.ClassName));
+            }
```

Comment:
```suggestion
            var callingConventionSyntax = (FunctionPointerUnmanagedCallingConventionSyntax)syntax;
            if (callingConventionSyntax.IsSpecialUnmanagedCallingConvention() ||
                semanticModel.Compilation.UnmanagedCallingConventionType(callingConventionSyntax.Name.ValueText) != null);
            {
                result.Add(new ClassifiedSpan(callingConventionSyntax.Name.Span, ClassificationTypeNames.ClassName));
            }
```

Reply:
@CyrusNajmabadi This suggestion doesn't match the original intent.

I think we first need to settle on what behavior the IDE should have.